README.md

```diff
@@ -31,14 +31,20 @@
    ```
    GATSBY_SITE_PATH_PREFIX =
    GATSBY_GIT_CORP_TOKEN = 1cdba7077XXXXXXXXXXXXXXXX633c1
+
    # Repo URL in given format which you want to use to generate a microsite
+   # NOTE: for Enterprise instances, this MUST be the FULL HTTPS URL
+   #       This repository must contain a manifest*.json file for Parliament to read.
```

Comment:
manfiest file can be a json or yaml, we recommend using yaml https://developers.corp.adobe.com/parliament-docs/onboarding.md#1-manifest-file-for-site-navigation

Reply:
updated